packages/playground/docs/getting-started-nested.md

```diff
@@ -7,32 +7,92 @@ permalink: /playground/
 
 # Getting Started
 
-The UI5 Web Components carry fundamental UI5 qualities, along with others, to provide enterprise-grade features, Fiori UX, and themeability. The goal is to achieve easy consumption of UI5 controls and to lower the entry barrier to use UI5 controls for own applications. By making the consumption of the full-stack UI5 optional, UI5 Web Components allow you to avoid the complexity of the UI5 rendering stack. In general, the UI5 Web Components are targeting web developers who want to have more flexibility to use just HTML tags or arbitrary JS frameworks.
+The UI5 Web Components share fundamental UI5 qualities with others to provide enterprise-grade features, Fiori UX and themeability. The goal is to achieve an easy consumption of UI5 controls to lower the entry barrier to use UI5 controls for own applications and to avoid complexity of the UI5 rendering-stack by making the consumption of the full-stack UI5 optional. In general, the UI5 Web Components target web developers who want to have more flexibility to use just HTML tags or arbitrary JS frameworks.
```

Comment:
1st sentence: **UI5 Web components** share....
2nd sentence: The goal of the **project/library/UI5 Web Components** is to achieve ... 

I know that this paragraph was here for a while now, but I think @LilyanaOviPe should take a closer look at it.

Reply:
1st sentence: I agree, should start with **UI5 Web Components** without 'The'. 
Also, it's not clear what these 'others' are, other controls, other components? And what are these qualities? It's better to be more specific here since this is the introductory sentence. Start with answering the question what are UI5 Web Components. Then you can explain their features. 
2 sentence: The UI5 Web Components project aims to achieve easier consumption of UI5 controls with lower entry barrier to the use of UI5 controls for own applications and less complexity of the UI5 rendering-stack by making the utilization of the full-stack UI5 optional. UI5 Web Components target web developers who want to have more flexibility to use just HTML tags or arbitrary JS frameworks.

Reply:
I only removed "The" in my first change and then it returned with the conflict resolution. Should be gone, agreed.

@LilyanaOviPe The whole sentence is left as it was in the previous edit. I also don't like the wording, could you change it as you see fit directly?

Reply:
@LilyanaOviPe Sorry, the resolve was mixed up. I'll add the sence like your edit left it and remove the first "The" as I intended.

Reply:
@pskelin Here is a proposal for the whole first paragraph, you can decide if it's OK:

UI5 Web Components are framework-independent UI elements incorporating the SAP Fiori design. UI5 Web Components share the fundamental enterprise-grade features of UI5 such as stability, accessibility and theming support. 
The UI5 Web Components project aims to achieve easier consumption of UI5 controls, lower entry barrier to UI5 controls use for own applications and less complexity of the UI5 rendering-stack by making the utilization of the full-stack UI5 optional. UI5 Web Components target web developers who want to have more flexibility to use just HTML tags or arbitrary JS frameworks.

Reply:
thanks, I took this version, much improved.